wordpress-mobile/WordPress-Android
# [Home Page Picker] Adds sample URL to the Choose a Domain Screen

**Fixes** https://github.com/wordpress-mobile/gutenberg-mobile/issues/3019

## Description
This PR enhances the Domain Search screen with a sample URL view. 

## To test:

1. Start the site creation flow (e.g. *Choose site > Add button*)
1. Select the *Create WordPress.com site* option
1. Press **Skip** or select a domain and press **Choose**
1. Verify that the sample URL view is shown

## Screenshots

|Phone Portrait|Phone Landscape|
|---|---|
|![mp](https://user-images.githubusercontent.com/304044/107641143-e72a0500-6c7b-11eb-87ae-4772429ac832.png)|![ml](https://user-images.githubusercontent.com/304044/107640841-87335e80-6c7b-11eb-9016-a6d7602863f5.png)|

|Tablet Portrait|Tablet Landscape|
|---|---|
|![tp](https://user-images.githubusercontent.com/304044/107641041-c6fa4600-6c7b-11eb-81b6-7ef8d40f380c.png)|![tl](https://user-images.githubusercontent.com/304044/107640849-8a2e4f00-6c7b-11eb-860b-6f44a2f148fc.png)|

PR submission checklist:
